From 84e2c9793f9240cec297eef8bb38978aade1fbb1 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 20 Aug 2014 15:15:38 +0300
Subject: [PATCH 24/59] ARM: i.MX6: dts: add i2c1 status okay

Restore the i2c1 bus staus that has been deleted while refactoring.
It was the reason why em3027 stoped working.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6q-sbc-fx6.dts  |    4 ++++
 arch/arm/boot/dts/imx6q-sbc-fx6m.dts |    1 +
 2 files changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6q-sbc-fx6.dts b/arch/arm/boot/dts/imx6q-sbc-fx6.dts
index 33e4f33..6f6ad33 100644
--- a/arch/arm/boot/dts/imx6q-sbc-fx6.dts
+++ b/arch/arm/boot/dts/imx6q-sbc-fx6.dts
@@ -20,6 +20,10 @@
    compatible = "compulab,cm-fx6", "compulab,sbc-fx6", "fsl,imx6q";
 };
 
+&i2c1 {
+	status = "okay";
+};
+
 &usdhc3 {
 	status = "okay";
 };
\ No newline at end of file
diff --git a/arch/arm/boot/dts/imx6q-sbc-fx6m.dts b/arch/arm/boot/dts/imx6q-sbc-fx6m.dts
index 2282250..cf2a0eb 100644
--- a/arch/arm/boot/dts/imx6q-sbc-fx6m.dts
+++ b/arch/arm/boot/dts/imx6q-sbc-fx6m.dts
@@ -37,6 +37,7 @@
 
 
 &i2c1 {
+	status = "okay";
 	rtc@56 {
 		compatible = "emmicro,em3027";
 		reg = <0x56>;
-- 
1.7.9.5

